### DISCUSSION TITLE
Occlusion Culling Help

### DISCUSSION THREAD

**Discussion by Diejkstra** [blackears/cyclopsLevelBuilder#179] [Q&A]
Does anyone have any tips on how to get the OccluderInstance3D node to recognize the meshes generated by Cyclops? I'm not sure if there is a setting I'm missing or if my node setup is incorrect, but it does correctly recognize any mesh not made through the plugin. The masks are set correctly as far as I know. Any guidance would be appreciated, thanks!

After baking the occluder, I moved the node upwards to show that it recognizes other meshes. 
![Untitled](https://github.com/blackears/cyclopsLevelBuilder/assets/33533531/e5b98dc0-c57b-4c15-97d6-5fe2c5974bd9)

**Comment by Diejkstra** [comment]
I just saw an open issue on this on the last page, that's my bad.

**Comment by blackears** [comment]
For adding occlusion culling, I'd recommend exporting your scene to regular Godot objects and then baking the exported scene.  It will probably not work on CyclopsBlock directly due to how Godot treats `owner` objects.
